packages/editor-ui/src/App.vue

```diff
@@ -254,7 +254,7 @@ export default defineComponent({
 			this.postAuthenticateDone = true;
 		},
 	},
-	async mounted() {
+	async created() {
```

Comment:
I think this change is not intended, right?

Reply:
It's not. I got a revert commit coming in just wanted to make sure e2e tests are passing right now.